kubernetes/k8_create.yml.erb

```diff
@@ -0,0 +1,60 @@
+# Deployment for create and redis service
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: create-deployment
+  labels:
+    app: create
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: create
+  template:
+      metadata:
+        labels:
+          app: create
+      spec:
+        containers:
+          - name: create-worker
+            image: "docker.ocf.berkeley.edu/create:<%= version %>"
+            imagePullPolicy: Always
+            ports:
+              - containerPort: 6378
+            resources:
+              limits:
+                cpu: 750m
+                memory: 512Mi
+            volumeMounts:
+              - mountPath: /home
+                name: nfs-export-home
+              - mountPath: /services
+                name: nfs-export-services
```

Comment:
mentioned it in the puppet PR, but is this actually needed?

Reply:
Yes, see comment in other pr